setup.py

```diff
@@ -365,7 +365,7 @@ def do_setup():
             'cryptography>=0.9.3',
             'dill>=0.2.2, <0.4',
             'flask>=1.1.0, <2.0',
-            'flask-appbuilder>=1.12.5, <2.0.0',
+            "flask-appbuilder>=2.2.0, <2.3.0",
```

Comment:
```suggestion
            'flask-appbuilder>=2.2.0, <2.3.0',
```

Reply:
Also is there a reson we should limit it to 2.3, Would `<3`, or `~=2.2` be better? https://www.python.org/dev/peps/pep-0440/#compatible-release